app/assets/javascripts/controllers/buttons/paging_div_button_group_controller.js

```diff
@@ -0,0 +1,66 @@
+ManageIQ.angular.app.controller('pagingDivButtonGroupController', ['$scope', 'miqService', '$compile', '$attrs', function($scope, miqService, $compile, $attrs) {
+  var init = function() {
+    if ($attrs.pagingDivButtonsType == "Submit") {
+      saveButton('Submit');
+      cancelButton();
+    } else {
+      saveButton();
+      resetButton();
+      cancelButton();
+    }
+
+    $scope.saveable = miqService.saveable;
+    $scope.disabledClick = miqService.disabledClick;
+  }
+
+  var saveButton = function(type) {
+    altText =  __("Save changes");
+    btnText = __("Save");
+
+    if(type == "Submit") {
+      altText =  __("Submit changes");
+      btnText = __("Submit");
+    }
+
+    var disabledSaveHtml = sprintf('<button name="button" id="save_disabled" type="submit" class="btn btn-primary btn-disabled" ' +
+      'alt=%s title=%s ng-click="disabledClick($event)" style="cursor:not-allowed" ' +
```

Comment:
Might want to replace `alt=%s` with something like `alt="{{ something }}"` and inject it into the $scope you're passing in. (Or create a new scope that inherits from this one.) - So that we don't get nasty surprises should that string contain something escape-worthy.